plugins/modules/azure_rm_aks_info.py

```diff
@@ -90,7 +91,7 @@ def __init__(self):
         self.module_args = dict(
             name=dict(type='str'),
             resource_group=dict(type='str'),
-            tags=dict(type='list'),
+            tags=dict(type='list', elements='str'),
```

Comment:
All added `elements` need a changelog fragment, since these potentially change behavior. (That's why I would avoid them for this collection, it's a lot of work for something that's dead anyway :) )

I think the change for this module is fine though.

Reply:
sounds sensible, done, thanks!